bgpd/bgp_route.c

```diff
@@ -8175,7 +8175,7 @@ DEFUN (show_ip_bgp_ipv4,
 
 ALIAS (show_ip_bgp_ipv4,
        show_bgp_ipv4_safi_cmd,
-       "show bgp ipv4 (unicast|multicast) {json}",
+       "show bgp ipv4 "BGP_SAFI_CMD_STR" {json}",
        SHOW_STR
        BGP_STR
        "Address family\n"
```

Comment:
Again.  All places that we modify (unicast|multicast) -> "BGP_SAFI_CMD_STR" need to be adjusted as well.  Adding a commit at the end would be fine imo